source/common/http/conn_manager_impl.cc

```diff
@@ -204,6 +204,7 @@ void ConnectionManagerImpl::doDeferredStreamDestroy(ActiveStream& stream) {
   stream.filter_manager_.disarmRequestTimeout();
 
   stream.completeRequest();
+  stream.filter_manager_.preDestroyFilters();
```

Comment:
Can you be consistent here with naming?

Reply:
sorry about that, missed it in the search and replace